modules/ROOT/pages/migrate-cl.adoc

```diff
@@ -1,49 +1,51 @@
+* The `autologin` kernel command-line parameter is not currently supported in FCOS.
+* Certain CL platforms, such as Vagrant, are not yet supported in FCOS. Refer to the https://getfedora.org/coreos/download/[Download page] to see a list of the available image types.
 
 == Configuration changes
-The xref:fcct-config.adoc[Ignition specification] has changed in both syntax and function. As a result, consider the following when you write new FCC files:
+
+When writing FCC files, keep the following changes in mind:
 
 * Users and groups are defined in the xref:ign-passwd.adoc[`passwd`] node of the FCC file.
-//* CLC --> FCC changes. Need more info.
-//* coreos-metadata rename - need more info.
-//* PXE custom OEMs - need more info.
+* `coreos-metadata` is now https://github.com/coreos/afterburn/blob/master/README.md[Afterburn]. You will have to evaluate the environment variables in your systemd unit files. For the sake of comparison, you can refer to the https://github.com/coreos/afterburn/blob/master/docs/container-linux-legacy.md[legacy documentation].
+* PXE images can be customized with a bundled Ignition config. Follow the xref:bare-metal.adoc[PXE live boot] instructions.
 * Use https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/system_administrators_guide/sect-using_chrony[`chronyd`] rather than `ntpd` or `systemd-timesyncd`.
 * `etcd` is not included in FCOS. Refer to xref:running-containers.adoc[Running etcd] for instructions to run it as a container on FCOS.
-* flannel is not supported. `fcct` should support the configuration you need without the use of flannel.
-* The functionality of the reboot manager (`locksmith`) is rolled into https://github.com/coreos/zincati/blob/master/README.md[`zincati`].
-* Along with Fedora 31, FCOS does not allow SSH password logins by default. For full details on how to set up users, refer to the documentation for the xref:ign-passwd.adoc[`passwd`] node of the FCC file.
-//* docker version selection
-* Because `usermod` is not yet fully-functional on FCOS, there is a `docker` group in the `/etc/group` file. This is a stop-gap measure to facilitate a smooth transition to FCOS. The team is working on a more functional `usermod`, at which time the `docker` group will no longer be included by default.
-* There is no way to create directories below the `root` directory. Changes are restricted to `/etc` and `/var`. Refer to the documentation for the xref:ign-storage.adoc[`storage`] node of the FCC configuration for details about writing directories and files to FCOS.
+* flannel is not supported on FCOS.
+* The functionality of the reboot manager (`locksmith`) is rolled into https://github.com/coreos/zincati/blob/master/README.md[Zincati].
+* Be default, FCOS does not allow SSH password logins. For full details on how to set up users, refer to the documentation for the xref:ign-passwd.adoc[`passwd`] node of the FCC file.
```

Comment:
`By default`

The second sentence isn't really relevant to the first.  It'd be helpful to describe how to re-enable password logins, but excitingly, there's not a good way to do it right now.  Maybe we should just drop the second sentence.

Reply:
I meant to point to the login authentication options available in the `passwd` node: a password hash or ssh key. I will rephrase to explain that better.